processor/src/main/java/org/infinispan/protostream/annotations/impl/processor/MarshallerSourceCodeGenerator.java

```diff
@@ -176,6 +176,7 @@ private void generateMessageMarshaller(ProtoMessageTypeMetadata pmtm) throws IOE
          iw.printf("import %s;\n", toImportName != null ? toImportName : toImport.getName());
          iw.println();
       }
+      iw.print("import org.infinispan.protostream.*;");
```

Comment:
the import is not required if using the `var` keyword.

Reply:
Sounds good 👍